app/views/teaching_events/show/_get-into-teaching-event-summary.html.erb

```diff
@@ -1,4 +1,4 @@
-<div class="about-ttt-event">
+<div class="about-git-event">
   <p>Watch presentations, ask questions, speak to advisers one-to-one.</p>
```

Comment:
Is this used anywhere @ethax-ross - not sure it is?

If not, something to clear up post go-live?